build.properties

```diff
@@ -44,3 +44,4 @@ jmeter.run.port=8080
 
 izpack.version=5.0.7-RC1
 pentaho-eula-wrap-config.version=8.0-SNAPSHOT
+dependency.kafka-clients.version=0.10.2.1
```

Comment:
I know, but build team wants to transition everything away from that format.  So when we introduce new properties, we should use artifact.version